assets/js/modules/optimize/common/amp-experiment-json-field.test.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * AMPExperimentJSONField component tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import AMPExperimentJSONField from './amp-experiment-json-field';
+import { render } from '../../../../../tests/js/test-utils';
+import { STORE_NAME } from '../datastore/constants';
+import { STORE_NAME as CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as MODULE_ANALYTICS } from '../../analytics/datastore/constants';
+
+const allParamsRegistry = ( registry ) => {
```

Comment:
Since none of these are reused, it makes the tests harder to read by putting these functions above.

Please move the functions into their respective tests. The usage is a bit simpler then as well as it can always be `setupRegistry`.

Reply:
Done